src/System.Collections.Immutable/tests/ImmutableListBuilderTest.cs

```diff
@@ -178,6 +178,17 @@ public void Remove()
         }
 
         [Fact]
+        public void RemoveAllBugTest()
+        {
+            var builder = ImmutableList.CreateBuilder<int>();
+            var elemsToRemove = new[]{0, 1, 2, 3, 4, 5}.ToImmutableHashSet();
+            foreach(var elem in new[]{0, 1, 2, 3, 4, 5, 6})
+                builder.Add(elem);
```

Comment:
Is this important enough to try to find that case?  If so I can put in the time to do it.  It didn't seem important enough to me at the moment.